source/localizable/tutorial/ember-data.md

```diff
@@ -102,5 +102,5 @@ When we call `findAll`, Ember Data will attempt to make a GET request to `/renta
 
 You can read more about Ember Data in the [Models section](../../models/).
 
-Since we're using Mirage in our development environment, Mirage will return the data we've provided.
-When we deploy our app to a production server, we will need to provide a backend for Ember Data to communicate with.
+Since we've already set up Ember Mirage in our development environment, Mirage will return the data we requested.
+When we deploy our app to a production server, we will likely want to provide a backend server for Ember Data to communicate with to store and retrieve persisted data from.
```

Comment:
...for storing and retrieving persisted data. Anything saved only in Ember Data's local store will be lost when we close our app.